source/type/type.cpp

```diff
@@ -176,26 +176,48 @@ bool StopTime::is_valid_day(u_int32_t day, const bool is_arrival, const RTLevel
 }
 
 const StopTime* StopTime::get_base_stop_time() const {
-    const nt::StopTime* st_base = nullptr;
-    size_t nb_st_found = 0;
-    if (vehicle_journey == nullptr) { return nullptr; }
+    
+    if (vehicle_journey == nullptr) { 
+        return nullptr; 
+    }
+
     auto base_vj = vehicle_journey->get_corresponding_base();
-    if (base_vj == nullptr) { return nullptr; }
+    
+    if (base_vj == nullptr) { 
+        return nullptr; 
+    }
 
-    if (vehicle_journey == base_vj) { return this; }
+    if (vehicle_journey == base_vj) { 
+        return this; 
+    }
+    
+    std::vector<const nt::StopTime*> st_base;
     for (const auto& st_it: base_vj->stop_time_list) {
         if (stop_point == st_it.stop_point) {
-            st_base = &st_it;
-            ++nb_st_found;
+            st_base.push_back(&st_it);
         }
     }
-    //TODO handle lolipop vj, bob-commerce-commerce-bobito vj...
-    if (nb_st_found != 1) {
+
+    if (st_base.empty()) { return nullptr; }
+
+    // handle lolipop 
+    size_t nb_rt_st_found = 0;
+    if (st_base.size() > 1) {
```

Comment:
Ideas:
* Count the number of stops on each VJ and compare could be nice too (if they are not the same...)
* Mention that Levenshtein is probably the best idea to improve that in a comment
* Check information from Kirin's feed (if available) of what's added/removed